policyengine_us/variables/gov/states/co/tax/income/credits/cdcc/co_cdcc.py

```diff
@@ -17,7 +17,6 @@ def formula(tax_unit, period, parameters):
         # follow 2022 DR 0347 form and its instructions (in Book cited above):
         p = parameters(period).gov.states.co.tax.income.credits
         fed_agi = tax_unit("adjusted_gross_income", period)  # Line 4
-        itaxbc = tax_unit("income_tax_before_credits", period)  # Line 6
```

Comment:
@martinholmer this did not seem to be use in the computation, just confirming that this was intended

Reply:
@PavelMakarchuk,  Yes you can delete the `itaxbc` local variable because it is no longer needed after you added the `capped_cdcc` variable.   Thanks for cleaning this up.